contents/startups-clerky.mdx

```diff
@@ -0,0 +1,313 @@
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-spotlight.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-merch.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-launch.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-credit.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-heart.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/yc_logo.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/conceptvc_logo.jpg
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/better_capital_logo.jpg
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/boldstart_square.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/oliver-kicks.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-partner.png
+  - >-
+    https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-presenter.png
+---
+
+import { getImage, GatsbyImage } from 'gatsby-plugin-image'
+import { FeatureSnapshot } from 'components/FeatureSnapshot'
+export const refer = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/startup-refer.png"
+
+<div class="w-[calc(100%_+_4rem)] relative -mx-4 px-8 md:-mx-16 pt-32 pb-72 -mt-32 md:-mt-44">
+  <div class="md:hidden absolute top-0 left-0 w-full bottom-0 opacity-40">
+    <GatsbyImage image={getImage(props?.images[0])} alt="PostHog for startups" objectFit="cover" className="" />
+  </div>
+  <div class="hidden md:block absolute top-0 left-0 w-full bottom-0">
+    <GatsbyImage image={getImage(props?.images[1])} alt="PostHog for startups" objectFit="cover" className="" />
+  </div>
+  <div class="absolute bottom-0 left-4 w-24 md:left-16 md:w-48">
+    <GatsbyImage image={getImage(props?.images[2])} alt="PostHog for startups" objectFit="contain" className="" />
+  </div>
+  <div class="absolute bottom-0 -right-4 w-28 md:right-16 md:w-60">
+    <GatsbyImage image={getImage(props?.images[3])} alt="PostHog for startups" objectFit="contain" className="" />
+  </div>
+
+  <div class="absolute top-0 left-0 w-full h-24 bg-gradient-to-b from-tan/100 via-tan/85 to-tan/0"></div>
+
+  <div className="max-w-[849px] mx-auto text-center">
+  <h1 className="text-4xl md:text-[64px] leading-none mt-0 mb-5">
+  PostHog x Clerky
+  </h1>
+  <h2 className="text-[18px] md:text-[20px] leading-[1.4] font-semibold mb-8 !mt-5">Clerky users can claim $55,000 in PostHog credit,<br className="hidden lg:block" />plus other benefits to help them build better products</h2>
```

Comment:
Why don't we say like "$50k in startup credit + $5k extra thanks to Clerky" It makes partnerships feel more special.

Reply:
```suggestion
  <h2 className="text-[18px] md:text-[20px] leading-[1.4] font-semibold mb-8 !mt-5">Claim $50,000 in PostHog credit,<br className="hidden lg:block" />plus an extra $5,000 of credit thanks to Clerky!</h2>
```